cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -815,9 +815,13 @@ func newServiceAccountTokenControllerDescriptor(rootClientBuilder clientbuilder.
 func startServiceAccountTokenController(ctx context.Context, controllerContext ControllerContext, controllerName string, rootClientBuilder clientbuilder.ControllerClientBuilder) (controller.Interface, bool, error) {
 	logger := klog.FromContext(ctx)
 	if len(controllerContext.ComponentConfig.SAController.ServiceAccountKeyFile) == 0 {
-		logger.Info("Controller is disabled because there is no private key", "controller", controllerName)
```

Comment:
restore this info message, if the controller can't run because no signing key was provided, that continues to be important to surface

Reply:
ok, will restore it.